### PR TITLE
Added coverage for edge cases with the pagination section

### PR DESCRIPTION
**Pull request by Lightining1boy**

### CODE REVIEW

File: client/src/component/App.jsx [hunk 1]
```diff
@@ -106,32 +106,53 @@ class App extends React.Component {
...
         currentPage: currentPage - 1,
       });
     }
   }
 
-  forwardArrowClicked() {
-    const { splitReviews, currentPage } = this.state;
-    if (splitReviews[currentPage + 1] !== undefined) {
+  forwardArrowClicked(search) {
+    console.log(search);
```
Comment (sbtorres) [inline client/src/component/App.jsx]: Don't forget to remove the console logs!

File: client/src/component/Pages.jsx [hunk 2]
```diff
@@ -69,32 +69,83 @@ class Pages extends React.Component {
...
-            <PageButton type="button" onClick={forwardArrowClicked}>
+            <PageButton type="button" onClick={() => forwardArrowClicked(search)}>
+              {backArrow}
+            </PageButton>
+          </PageButtonList>
+        </PageButtonUL>
+      );
+    }
+    if (pageArray.length <= 4 && currentPage === 2) {
+      console.log('page 3');
```
Comment (sbtorres) [inline client/src/component/Pages.jsx]:
```suggestion
```

Comment (Lightining1boy) [reply client/src/component/Pages.jsx]: I'm so bad at getting rid of those!

Comment (Lightining1boy) [reply client/src/component/Pages.jsx]: I'll search through everything real quick and get rid of those

**Pull request merged**